### ISSUE TITLE
Disable horizontal scroll?

### ISSUE THREAD

**Issue by Bruno-Piccinin-RealComm** [BadChoice/RVCalendarWeekView#35]
It is possible to disable horizontal scroll (day scroll) and mantain the vertical scroll (hours scroll) on week view?

**Comment by BadChoice** [comment]
Yes, it is a collection view, so you can disable horitzontal scroll as you
would on a simple uicollectionview

Jordi Puigdellívol

El 29 de diciembre de 2017 a las 8:45:20, Bruno Piccinin (
notifications@github.com) escribió:

It is possible to disable horizontal scroll (day scroll) and mantain the
vertical scroll (hours scroll) on week view?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/BadChoice/RVCalendarWeekView/issues/35>, or mute the
thread

**Comment by Bruno-Piccinin-RealComm** [comment]
I tried to use

` _weekView.collectionView.alwaysBounceHorizontal = NO;
    _weekView.collectionView.alwaysBounceVertical = YES;`

in my setupWeekData (same as the example) but it doesn't work.
I found on internet something like this

`UICollectionViewFlowLayout *flowLayout = [[UICollectionViewFlowLayout alloc] init];
[flowLayout setScrollDirection:UICollectionViewScrollDirectionVertical];`

but I can't find a way to access on `UICollectionViewFlowLayout` of `weekView`, but I found only the `UICollectionViewLayout` with no possibilities to set the scroll direction
`_weekView.collectionView.collectionViewLayout`

A little help please?
